kyuubi-ha/src/main/scala/org/apache/kyuubi/ha/HighAvailabilityConf.scala

```diff
@@ -108,4 +109,20 @@ object HighAvailabilityConf {
     .version("1.4.0")
     .stringConf
     .createOptional
+
+  val HA_ZK_ENGINE_POOL_SIZE: ConfigEntry[Int] =
+    buildConf("ha.engine.pool.size")
+      .doc("Maximum number of engines when using USER share level.")
+      .version("1.4.0")
+      .intConf
+      .checkValue(s => s > 0 && s < 50, "Invalid Engine Pool Size.")
```

Comment:
`"Invalid Engine Pool Size."` => `"Invalid Engine Pool Size, should be between 0 and 50"`

Reply:
Good !!!